b2sdk/stream/progress.py

```diff
@@ -0,0 +1,59 @@
+from b2sdk.stream.wrapper import StreamWrapper
+
+
+class AbstractStreamWithProgress(StreamWrapper):
```

Comment:
moving classes can be easily split to another PR (then review is way easier as reviewer knows you were not messing with the code)

Reply:
yes - but I was having like 2k lines of changes - and mind set to solving a real problem - and such interrupt would cost my mind also significant amount of energy... also previous code was so buggy that I just wanted to remove it as quickly as possible...